src/Components/InputTodo.js

```diff
@@ -0,0 +1,89 @@
+import styled from 'styled-components';
+import { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { runToast } from './toast/Toast';
+
+const InputTodo = ({ list, setList }) => {
+    const [form, setForm] = useState('');
+    const handleChange = (e) => {
+        setForm(e.target.value);
+    };
+    const handleInput = () => {
+        const createItem = () => {
+            setList([
+                ...list,
+                {
+                    id: uuidv4(),
+                    content: form,
+                    done: false,
+                },
+            ]);
+        };
+
+        form // 입력한 값이 있고,
+            ? !list.find((item) => item.content === form) //중복되는 값이 없을 때
+                ? createItem()
+                : runToast('동일한 내용으론 작성할 수 없어요!')
+            : runToast('할 일을 입력해주세요!');
+
+        //입력창 초기화
+        setForm('');
+    };
+
+    const handleEnter = () => {
+        if (window.event.keyCode == 13) {
```

Comment:
[strict equality operator](https://developer.mozilla.org/ko/docs/Web/JavaScript/Equality_comparisons_and_sameness)를 사용하시는 게 좋습니다.

```suggestion
        if (window.event.keyCode === 13) {
```

`===`와 달리 `==`는 강제 type casting(형변환)을 적용하여 값을 비교합니다. 따라서 `===`을 사용하면 나중에 예상치 못한 버그가 발생했을 때 이를 발견하기가 더 쉽겠죠?